### DISCUSSION TITLE
I'm not using Firebase but "Missing project_info object" while build

### DISCUSSION THREAD

**Discussion by devaray** [expo/expo#18991] [Archived: Questions]
Hello,  I've a problem when i do expo build:android, it stops with this message: 

[stderr] Execution failed for task ':app:processReleaseGoogleServices'.

[stderr] > Missing project_info object

I'm not using firebase  but instead   @react-native-google-signin/google-signin and that is working very well. 
Anyone can help me with that problem.

**Comment by keith-kurak** [comment]
This error appears to be related to a missing or invalid google-services.json. I recommend double-checking the [configuration steps for the Google Sign-in package](https://github.com/react-native-google-signin/google-signin), which require this file. Additionally, it's not clear to be that this package would work with the Classic Build service, and classic builds are only available through the end of the year, so I would recommend moving over to [EAS Build](https://docs.expo.dev/build/introduction/).

**Reply by devaray** [reply]
Thank you for your reply.  Now I'm using EAS Build, and replaced my google-services.json  (provided by google console)  for provided for firebase console (Although I'm not using firebase). The Warning now is gone.
